### PR TITLE
Feature/inline errors

### PR DESCRIPTION
**Pull request by jacquesporveau**
https://www.pivotaltracker.com/story/show/140356551

We want to display errors in-line. I made a PR to do another way and SimpleForm was suggested. Sounds like we use that on a bunch of the projects we have going now so I decided to go with it.

### CODE REVIEW

File: app/views/presentations/_form.html.erb [hunk 1]
```diff
@@ -1,36 +1,15 @@
...
-    <%= f.label :duration %><br>
-    <%= f.text_field :duration %> Minutes
-  </p>
-
-  <p>
-    <%= f.label :event %><br>
-    <%= f.collection_select(:event_id, Event.upcoming, :id, :name, prompt: true) %>
-  </p>
+  <%= f.input :topic %>
+  <%= f.input :description, as: :string %>
```
Comment (qr8r) [inline app/views/presentations/_form.html.erb]: do you need the as here? I would have expected simple form to have figured that on on its own

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -1,23 +1,8 @@
...
-    <%= f.label :date %><br>
-    <%= f.date_field :date %>
-  </p>
-
-  <p>
-    <%= f.submit %>
-  </p>
+  <%= f.input :name %>
+  <%= f.input :location %>
+  <%= f.input :date, as: :date %>
```
Comment (qr8r) [inline app/views/events/_form.html.erb]: do you have to specify here?

File: app/views/people/_form.html.erb [hunk 3]
```diff
@@ -1,43 +1,12 @@
...
-    <%= f.submit %>
-  </p>
+<%= simple_form_for @person do |f| %>
+
+  <%= f.input :first_name %>
+  <%= f.input :last_name %>
+  <%= f.input :email %>
+  <%= f.input :password %>
+  <%= f.input :password_confirmation %>
+  <%= f.input :date_of_birth, as: :date %>
```
Comment (qr8r) [inline app/views/people/_form.html.erb]: Same

**Review by qr8r — COMMENTED** [review]
@jacquesporveau Just had some questions about how you are using simpleform.

I kind of expected it to infer a lot of the input types you are specifying

File: app/views/presentations/_form.html.erb (hunk 1 above)
Comment (jacquesporveau) [reply app/views/presentations/_form.html.erb]: This is actually a binary field so if I don't have as string is errors out.

**Comment by jacquesporveau** [comment]
Requested changes made @brendandeere

File: app/views/presentations/_form.html.erb (hunk 1 above)
Comment (natyork) [reply app/views/presentations/_form.html.erb]: That makes sense to me from the form perspective, but do you happen to know why we specified presentations description as a binary in our schema?

**Review by natyork — COMMENTED** [review]
No changes from me - simple_form seems really interesting, good job getting it implemented. 

I just had the one question as to whether you happen to know why we defined presentation description as a binary in our schema?

**Comment by jacquesporveau** [comment]
@natyork Descriptions might be long and I think the string field is limited to 255 char.

File: app/views/presentations/_form.html.erb (hunk 1 above)
Comment (jacquesporveau) [reply app/views/presentations/_form.html.erb]: Descriptions might be long and I think the string field is limited to 255 char.

**Review by qr8r — APPROVED** [review]

**Pull request merged**